experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -342,14 +343,12 @@ function get_atmos_config_dict(coupler_dict::Dict, job_id::String, atmos_output_
         error("Invalid `atmos_config_repo`; please use \"ClimaCoupler\" or \"ClimaAtmos\"")
     end
 
-    # use coupler toml if atmos is not defined
-    atmos_toml_file = atmos_config["toml"]
-    coupler_toml_file = coupler_dict["coupler_toml_file"]
+    # use atmos toml if coupler toml is not defined
```

Comment:
This is the logic we had in the code before, but the comment didn't match what the code was doing